crates/bevy_gizmos/src/aabb.rs

```diff
@@ -96,7 +97,7 @@ fn draw_all_aabbs(
     }
 }
 
-fn color_from_entity(entity: Entity) -> LegacyColor {
+fn color_from_entity(entity: Entity) -> Color {
```

Comment:
Since this method generates a nice sequence of colors based on the `u32` representation of an `Entity`, maybe we could offer a more general `pub fn color_from_index(index: u32) -> Oklcha`? Feels like a pretty handy function for debugging purposes, and might be nice for end users to have access to a color palette that can be programmatically accessed.

Reply:
Spun out into #12170.